docs/release-notes/fetching-metrics-fix.rst

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Bug Fixes**
+
+-  WebUI: Fix endless metrics fetching for cancelled experiments, without metrics, on the
```

Comment:
"Fix endless metrics fetching in "Visualization" tab in experiment details page for cancelled experiments that do not have metrics."

Reply:
Thanks! Updated.